config/routes.rb

```diff
@@ -69,6 +69,11 @@
     get('/summary/allocated' => 'summary#allocated')
     get('/summary/unallocated' => 'summary#unallocated')
     get('/summary/pending' => 'summary#pending')
+
+    resources :offenders, only: [] do
```

Comment:
Do we definitely need another version of /prisoners?

Reply:
no - we didn't have a resource for it. We do now :-)

Reply:
Probably, yes.